specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+        }
+      }
+    },
+    "ProvisioningServiceErrorDetails": {
+      "description": "Contains the properties of an error returned by the Azure IoT Hub Provisioning Service.",
+      "type": "object",
+      "properties": {
+        "errorCode": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "trackingId": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "info": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "timestampUtc": {
+          "format": "date-time",
+          "type": "string"
+        }
+      }
+    },
+    "DeviceOperationStatusResponse": {
+      "type": "object",
+      "properties": {
+        "operationId": {
+          "description": "An opaque operation ID string.",
+          "type": "string",
+          "readOnly": true,
+          "example": "IjAwMDBlOGMyLTAwMDAtMDYwMC0wMDAwLTYxOGVkODYxMDAwMCI="
+        },
+        "deviceOperationStatus": {
+          "description": "Device operation status.",
+          "enum": [
+            "unknown",
+            "notStarted",
+            "running",
+            "failed",
+            "succeeded"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "example": "running",
+          "x-ms-enum": {
+            "name": "deviceOperationStatus",
+            "modelAsString": true
```

Comment:
Normally, we do recommend modelAsString true like you've done here, but if you're 100% sure the enum values won't change - which is typical of status enum values - this can be false (or elided). You're call. modelAsString is safe, but tends to have downstream ramifications for SDKs like .NET that turn these into enum-like strings.

Reply:
I can assure you that the enum values will not change. We have checks in place that fail a checkin if this were to be changed. Furthermore, since there isn't an older API to compare against in this repo, here is what the diff looks like for another enum type that is already GA.

<img width="986" alt="image" src="https://user-images.githubusercontent.com/22642794/172019857-46c2c80e-62ee-46db-b53c-221bfa6f0850.png">


In our review if you recall, I brought up my concerns of any backwards compat, hopefully this diff reflects my concerns and it would be good to get an ACK.